content/en/actions/app_builder/build.md

```diff
@@ -37,9 +37,32 @@ Blueprints are helpful starter apps that cover common use cases. They come loade
 1. Each blueprint template comes loaded with demo data. To customize the app, edit the **Connection** for each query.
 1. To save the app, click **Save as New App**.
 1. To preview the app, click **Preview**. Click **Edit** from the preview screen to return to the configuration view.
-1. After you finish modifying the app, Click **Run** to test it.
 1. When you're ready to publish your app, click **Publish**. Publishing an app makes it available to your dashboards.
 
+### Build an app with AI
+
+With Bits AI, you can generate a complete app from a single prompt. Describe the app you want to create, and the agent will automatically generate the UI, actions, and logic mapped to your existing data and permissions. You can then iterate by chatting---refining components, flows, or styling---all without writing any code.
```

Comment:
```suggestion
With Bits AI, you can generate a complete app from a single prompt. Describe the app you want to create, and the agent automatically generates the UI, actions, and logic mapped to your existing data and permissions. You can then iterate by chatting---refining components, flows, or styling---without writing any code.
```

Reply:
I'm guessing `agent` here does not refer to the Datadog Agent. Would it be possible to disambiguate this, as references to the Datadog Agent are widespread in the docs?